locale/pt-br/blog/video/bert-belder-libuv-lxjs-2012.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Bert Belder - libuv na LXJS 2012
+slug: bert-belder-libuv-lxjs-2012
+category: video
+date: 2012-09-30T17:28:45.000Z
+layout: blog-post.hbs
+---
+
+Bert Belder, Desenvovedor do núcleo Node, deu uma palestra no
```

Comment:
```suggestion
Bert Belder, Desenvovedor do core do Node, deu uma palestra no
```

I'd rather not translate "core" into "núcleo" because no dev actually talks like that here

Reply:
I will do it, thanks